bpdm-pool/pom.xml

```diff
@@ -130,6 +130,11 @@
             <groupId>com.fasterxml.jackson.module</groupId>
             <artifactId>jackson-module-kotlin</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.hibernate.orm</groupId>
+            <artifactId>hibernate-jpamodelgen</artifactId>
+            <version>${hibernate.version}</version>
+        </dependency>
```

Comment:
Sorry guys, I misunderstood where this functionality comes from. Direct Hibernate dependencies are problematic and if we have a good alternative we should try to avoid them. Hibernate dependencies run under the LGPL 2.1 license. We should avoid such licenses whenever possible. So if you can implement the functionality without this dependency this would be preferable.

Reply:
I removed the dependency and changed the implementation to field reflection.